ghost/core/test/unit/frontend/helpers/concat.test.js

```
@@ -87,4 +90,63 @@ describe('{{concat}} helper', function () {
         let expected = '[object Object]?my=param';
         shouldCompileToExpectedWithGlobals(templateString, {}, expected, defaultGlobals);
     });
+
+    it('can concat empty safestrings', function () {
+        compile('{{concat non_empty_safe_string empty_safe_string}}')
```

ESLint

[error] 'compile' is not defined.

(no-undef)